packages/dds/sequence/src/test/intervalCollection.spec.ts

```diff
@@ -848,21 +848,9 @@ describe("SharedString interval collections", () => {
 		});
 
 		it("propagates delete op to second runtime", async () => {
-			containerRuntimeFactory = new MockContainerRuntimeFactoryForReconnection();
-
-			// Connect the first SharedString.
-			const containerRuntime1 =
-				containerRuntimeFactory.createContainerRuntime(dataStoreRuntime1);
-			const services1: IChannelServices = {
-				deltaConnection: dataStoreRuntime1.createDeltaConnection(),
-				objectStorage: new MockStorage(),
-			};
-			sharedString.initializeLocal();
-			sharedString.connect(services1);
-
 			// Create and connect a second SharedString.
 			const runtime2 = new MockFluidDataStoreRuntime();
-			const containerRuntime2 = containerRuntimeFactory.createContainerRuntime(runtime2);
+			containerRuntimeFactory.createContainerRuntime(runtime2);
```

Comment:
this test suite sets up `containerRuntimeFactory` and `sharedString1` in a beforeEach hook. So runtimes were getting reused/not hooked up appropriately. I didn't dig super deep into this, but it looked like ops were not getting correctly sent due to this bad setup and that tripped up some of the added consistency asserts in sequence.ts. Test passed before since "no sent ops" makes an empty interval collection in addition to "add an interval and remove it".